crates/rooch-types/src/framework/account_authentication.rs

```diff
@@ -81,6 +85,22 @@ impl<'a> AuthenticationKeyModule<'a> {
             result.into()
         })
     }
+
+    pub fn rotate_authentication_key_action<V: MoveStructType>(public_key: Vec<u8>) -> MoveAction {
```

Comment:
`rotate_authentication_key` function in account_authentication.move is not a entry function, so it can not called via the MoveAction.

Reply:
Moved to validators Rust Module.